src/main/java/com/strandls/esmodule/controllers/ESController.java

```diff
@@ -389,6 +389,50 @@ public MapDocument termsAggregation(@PathParam("index") String index, @PathParam
 		}
 	}
 
+	@GET
+	@Path(ApiConstants.AGGREGATION + "/{filter}")
+	@Produces(MediaType.APPLICATION_JSON)
+
+	@ApiOperation(value = "Aggregation for List Page", notes = "Returns Aggregated values", response = Map.class)
```

Comment:
give meaningful value and notes , it will be useful when using swagger-ui and also other devs would understand this easily.